google/cloud/bigquery_unified/BUILD.bazel

```diff
@@ -53,6 +60,15 @@ cc_library(
     name = "google_cloud_cpp_bigquery_bigquery_unified",
     srcs = google_cloud_cpp_bigquery_bigquery_unified_srcs,
     hdrs = google_cloud_cpp_bigquery_bigquery_unified_hdrs,
+    # Enable the config when the bool flag is visible.
```

Comment:
same, although it will be unblocked probably tomorrow

Reply:
Yeah, added https://github.com/googleapis/google-cloud-cpp-bigquery/issues/98